src/rss-diff.js

```diff
@@ -0,0 +1,5 @@
+function rssNDiff(itemBefore, itemAfter, field) {
+    let delta
```

Comment:
Incomplete? We should probably add eslint-standard to prevent things like this from getting checked in (see https://github.com/windbreaker-io/windbreaker-service-util/blob/master/package.json#L53) and add an .eslintrc file to extend standard in the project root.